lms/djangoapps/branding/api.py

```diff
@@ -329,3 +330,63 @@ def _absolute_url_staticfile(is_secure, name):
     # For local development, the returned URL will be relative,
     # so we need to make it absolute.
     return _absolute_url(is_secure, url_path)
+
+
+def get_microsite_url(name):
+    """
+    Look up and return the value for given url name in microsite configuration.
+    Return '#' if given url name is not defined in microsite configuration.
```

Comment:
Here we should say "Return EMPTY_URL", because we are not sure what it contains (#)